app/models/document.rb

```diff
@@ -4,6 +4,13 @@ class Document < ::PgSearch::Document
   scope :featureable, -> { where(searchable_type: ['Journey', 'App', 'Page']) }
   scope :featured, -> { featureable.where(featured: true) }
 
+  scope :join_on_category_id, -> (category_id) {
```

Comment:
Toto by sa mozno dalo zapisat krajsie (viac railsy way) ale inak ok. V kazdom pripade toto vyzera skor ako filter na category_id cize by som to nazval `scope: with_category, -> (category)` a nepchal tam surove id pokial mame aj objekt (primitive obsession code smell)